src/BBT.Maybe/MaybeStruct.cs

```diff
@@ -15,84 +15,84 @@
     public struct MaybeStruct<T> : ISerializable, IEquatable<MaybeStruct<T>>
         where T : struct
     {
-        private readonly T? mValue;
+        private readonly T? value;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="MaybeStruct{T}"/> struct.
         /// </summary>
-        /// <param name="aValueNullable">The potentially nullable value.</param>
-        internal MaybeStruct(T? aValueNullable)
+        /// <param name="value">The potentially nullable value.</param>
+        internal MaybeStruct(T? value)
         {
-            this.mValue = aValueNullable;
+            this.value = value;
         }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="MaybeStruct{T}"/> struct.
         /// Constructor used for deserialization.
         /// </summary>
-        private MaybeStruct(SerializationInfo aInfo, StreamingContext aContext)
+        private MaybeStruct(SerializationInfo info, StreamingContext context)
         {
-            MaybeUtils.CheckArgumentNotNull(aInfo, nameof(aInfo));
-            this.mValue = MaybeUtils.GetDeserializedValue<T>(aInfo, nameof(this.mValue));
+            MaybeUtils.CheckArgumentNotNull(info, nameof(info));
+            this.value = MaybeUtils.GetDeserializedValue<T>(info, nameof(this.value));
         }
 
         /// <summary>
         /// Gets a value indicating whether <see cref="Maybe"/> has a value (true)
         /// or is representing the null case (false).
         /// </summary>
-        public bool HasValue => this.mValue.HasValue;
+        public bool HasValue => this.value.HasValue;
 
         /// <summary>
         /// Checks whether the operands are equal.
         /// </summary>
-        /// <param name="aA">Maybe to compare.</param>
-        /// <param name="aB">Maybe to compare.</param>
-        public static bool operator ==(MaybeStruct<T> aA, MaybeStruct<T> aB)
+        /// <param name="a">Maybe to compare.</param>
+        /// <param name="b">Maybe to compare.</param>
+        public static bool operator ==(MaybeStruct<T> a, MaybeStruct<T> b)
         {
-            return object.Equals(aA.mValue, aB.mValue);
+            return object.Equals(a.value, b.value);
         }
 
         /// <summary>
         /// Checks whether the operands are unequal.
         /// </summary>
-        /// <param name="aA">Maybe to compare.</param>
-        /// <param name="aB">Maybe to compare.</param>
-        public static bool operator !=(MaybeStruct<T> aA, MaybeStruct<T> aB)
+        /// <param name="a">Maybe to compare.</param>
+        /// <param name="b">Maybe to compare.</param>
+        public static bool operator !=(MaybeStruct<T> a, MaybeStruct<T> b)
         {
-            return !(aA == aB);
+            return !(a == b);
```

Comment:
`return !(a == b);` may be simplified by `return a != b;`

Reply:
Done.